tests/contrib/minikube/test_kubernetes_pod_operator.py

```diff
@@ -264,6 +264,27 @@ def test_volume_mount():
             k.execute(None)
             mock_logger.info.assert_any_call(b"retrieved from mount\n")
 
+    @staticmethod
+    def test_security_context():
```

Comment:
What's this testing ? should you assert that the security context is indeed modified in the KubernetesPodOperator that is created ?

I don't think executing is enough

Reply:
Right now it's testing that it doesn't throw an exception, but I agree, more tests would be useful.

Reply:
I agree. I mostly mimicked the other similar tests for pod attributes but the whole test suite is lacking a bit in this regard.